src/ddbToEs/ddbToEsHelper.ts

```diff
@@ -4,39 +4,18 @@
  */
 
 import { Client } from '@elastic/elasticsearch';
-// @ts-ignore
-import { AmazonConnection, AmazonTransport } from 'aws-elasticsearch-connector';
 import allSettled from 'promise.allsettled';
-import AWS from '../AWS';
 import PromiseParamAndId, { PromiseType } from './promiseParamAndId';
 import { DOCUMENT_STATUS_FIELD } from '../dataServices/dynamoDbUtil';
 import DOCUMENT_STATUS from '../dataServices/documentStatus';
 
 const BINARY_RESOURCE = 'binary';
 
-const { IS_OFFLINE, ELASTICSEARCH_DOMAIN_ENDPOINT } = process.env;
-
 export default class DdbToEsHelper {
     private ElasticSearch: Client;
 
-    constructor() {
-        let ES_DOMAIN_ENDPOINT = ELASTICSEARCH_DOMAIN_ENDPOINT || 'https://fake-es-endpoint.com';
-        if (IS_OFFLINE === 'true') {
-            const { ACCESS_KEY, SECRET_KEY, AWS_REGION, OFFLINE_ELASTICSEARCH_DOMAIN_ENDPOINT } = process.env;
-
-            AWS.config.update({
-                region: AWS_REGION || 'us-west-2',
-                accessKeyId: ACCESS_KEY,
-                secretAccessKey: SECRET_KEY,
-            });
-            ES_DOMAIN_ENDPOINT = OFFLINE_ELASTICSEARCH_DOMAIN_ENDPOINT || 'https://fake-es-endpoint.com';
-        }
-
-        this.ElasticSearch = new Client({
-            node: ES_DOMAIN_ENDPOINT,
-            Connection: AmazonConnection,
-            Transport: AmazonTransport,
-        });
+    constructor(ElasticSearch: Client) {
```

Comment:
Do we want to do this across the package? i.e. Deployment package initializes the AWS services and passes it down?

Reply:
I'd argue that we should not make this change as part of this PR. It's a bit too extreme and it's not strictly needed to fix the issue.

we could do something like:
```js
class ourNewClass {
  private ElasticSearch: Client;
  constructor(){
    this.elasticsearch = new Client()
  }
  handleDdbToEsEvent(event: any) {
    ...
  }
}
``` 
to bring it in line with all the other fwoa packages(they tend to init aws clients in constructors)

I think that later we can evaluate the option of having fwoa components accept aws clients as optional constructor params. IMO optional is better since customers often only want to pass down clients if they have a specific client config in mind (i.e. very specific retry policies) otherwise it's convenient to just use a default AWS client.